config/config.go

```diff
@@ -79,12 +92,14 @@ func defaultConfig() *Config {
 
 func DefaultConfigMainnet() *Config {
 	conf := defaultConfig()
-	conf.Network.DefaultBootstrapAddrStrings = []string{
-		"/dns/bootstrap1.pactus.org/tcp/21888/p2p/12D3KooWMnDsu8TDTk2VV8uD8zsNSB6eUkqtQs6ttg4bHq9zNaBe",
-		"/dns/bootstrap2.pactus.org/tcp/21888/p2p/12D3KooWM39ag7ghta49qybf7McADgT8FLakTYkCsiBvwdnjuG5q",
-		"/dns/bootstrap3.pactus.org/tcp/21888/p2p/12D3KooWBCPSZWheet6tMoHbVBCDfBwQm4yzCwcQ8hJ6NMCN97sj",
-		"/dns/bootstrap4.pactus.org/tcp/21888/p2p/12D3KooWKg6aLa77yAaqMCb45aH5iQuTr5GzRUWUCJ1sZYB5vnoL",
+
+	bootstrapNodes := make([]Bootstrap, 0)
+	if err := json.Unmarshal(bootstrapNodesBytes, &bootstrapNodes); err == nil {
```

Comment:
Please return err if there is any error here.
Otherwise we have empty list for Bootstrap without any error

Reply:
We handle reverse err == nil and never panic happen here